code/index.html

```diff
@@ -1,34 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./style.css" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet" />
-    <title>Chat Bot</title>
-  </head>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Cormorant+Infant&family=Montserrat:wght@400;500&display=swap"
+    rel="stylesheet">
+  <title>Tablebooker</title>
+  <script src="https://kit.fontawesome.com/123ae26214.js" crossorigin="anonymous"></script>
+</head>
 
-  <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+<body>
+  <div class="content-wrapper">
+    <h1>Tablebooker</h1>
     <main>
       <section class="chat" id="chat"></section>
-
       <div class="input-wrapper" id="input-wrapper">
-        <form id="name-form">
-          <label for="name-input">name</label>
-          <input id="name-input" type="text" />
-          <button class="send-btn" type="submit">
-            Send
-          </button>
+        <form id="input-form" action="#">
```

Comment:
Thank you for the feedback!

Yes exactly, the action = # makes the form submit to itself thus or preventing the page from reloading. So just like you said it pretty much does the same thing :)

Reply:
> Comment on "Action = '#'", is this used instead of "event.preventDefault();" in JS?

Yes, that’s right. Having it straight in the HTML-file instead of in the JS-file makes debugging easier since the page won't reload when submitting the form even if the JavaScript is broken.